### PR TITLE
Variable fonts article

### PR DESCRIPTION
**Pull request by mustafa-x**
What's changed, or what was fixed?
New article on variable fonts. 

**Target Live Date:** 2018-02-20

- [x] I have run `gulp test` locally and most tests pass, some links are quite long.
- [x] I've staged the site and manually verified that my content displays correctly.

### CODE REVIEW

File: npm-debug.log [hunk 1]
```diff
@@ -0,0 +1,37 @@
+0 info it worked if it ends with ok
```
Comment (petele) [inline npm-debug.log]: Please remove this file

File: src/content/en/fundamentals/design-and-ux/typography/variable-fonts/index.md [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+  </figure>
+</div>
+<div class="attempt-right">
+  <figure>
+    <img src="images/roboto-family.png" alt="Different members of the Roboto family">
+  </figure>
+</div>
+<div class="clearfix"></div>
+
+<div class="note"><strong>Note:</strong><span> Roboto Family designed by Christian Robertson.
```
Comment (petele) [inline src/content/en/fundamentals/design-and-ux/typography/variable-fonts/index.md]: The span isn't required here. If you wanted you could simply do:
```
Note: Roboto Family designed by Christian Robertson.
```

DevSite will automatically handle the formatting and such for you.

File: src/content/en/fundamentals/design-and-ux/typography/variable-fonts/index.md [hunk 3]
```diff
@@ -0,0 +1,314 @@
...
+On the web, we have both of the above for consideration, but also the associated
+bandwidth cost. For every member of the family or style used in our designs, we
+require that font file to be downloaded by the reader before the text is
+rendered. Just including the regular and bold styles plus their italic
+variations can amount to 500k or more of data. This has been a sticking point
+for web designers, and developers as a customized typography experience can come
+at a cost. This is even before we have dealt how the font renders and the
+patterns we are going to use (FOIT and FOUT).
+
+## Anatomy of a variable font A variable font is a collection of masters, with
```
Comment (petele) [inline src/content/en/fundamentals/design-and-ux/typography/variable-fonts/index.md]: The whole paragraph here likely is an H2, which I don't think is what you mean.

File: src/content/en/fundamentals/design-and-ux/typography/variable-fonts/index.md [hunk 4]
```diff
@@ -0,0 +1,314 @@
...
+each axis we get to explore multiple options of the font, not just the specific
+axes, but their combinations. For example, variable font Amstelvar has three
+masters for its <strong>Weight axis</strong>, Regular, which is at the center of
+the Weight axis scale, and two masters, thinner and bolder at the opposite ends
+of the axis. Between these, there are potentially 200 variations on the axis
+that give the designer or developer an array of weights to choose from and
+explore.
+
+[Amstelvar](https://github.com/TypeNetwork/Amstelvar){: .external} also has
+[three masters for its Width axis, Regular, which is at the center of the axis
```
Comment (petele) [inline src/content/en/fundamentals/design-and-ux/typography/variable-fonts/index.md]: All of these lines start with `[` - did you mean to do that?

File: src/content/en/fundamentals/design-and-ux/typography/variable-fonts/index.md [hunk 5]
```diff
@@ -0,0 +1,314 @@
...
+
+So among Amstelvar’s registered axes, width, weight and optical size, there are
+thousands of styles which may seem like massive overkill, and Amstelvar is only
+an example of a Latin typeface design. Considering the needs of all the world’s
+scripts, and for some of today’s typographic applications, quality of user
+experience can be remarkably enhanced by this diversity of type styles within a
+font. And, if it is without performance penalty, the user can use a few, or use
+many, it’s up to their design.
+
+### Italics are slightly different Type families like Helvetica or Roboto have
```
Comment (petele) [inline src/content/en/fundamentals/design-and-ux/typography/variable-fonts/index.md]: Again, whole paragraph is wrapped in an `h3`

File: src/content/en/fundamentals/design-and-ux/typography/variable-fonts/index.md (hunk 4 above)
Comment (mustafa-x) [reply src/content/en/fundamentals/design-and-ux/typography/variable-fonts/index.md]: oops good catch

**Comment by mustafa-x** [comment]
Changes made @petele  :)

**Comment by mustafa-x** [comment]
The issues that that come up here are from the master branch.

**Comment by petele** [comment]
Replaced by #5817

**Pull request closed without merging**